### DISCUSSION TITLE
How to enable Cloud Changes

### DISCUSSION THREAD

**Discussion by songoffireandice03** [VSCodium/vscodium#1531] [Q&A]
I can't seem to connect to my GitHub account, no matter how many times I pressed Turn On Cloud Changes. How can I sync VSCodium  to my GitHub?

OS version: Windows 11 21H2 22000.1936
VSCodium version: 1.78.2.23132

**Comment by Pdzly** [comment]
I have the same question. If i try "Activate Cloud Sync" nothing happens

**Comment by phenrique-coder** [comment]
I have the same problem.

**Comment by treevel** [comment]
I have the same problem.
OS version: Windows 10 22H2 19045.3324, macOS 12.6.9
VSCodium version: 1.82.2

**Comment by dallenbaldwin** [comment]
When attempting to enable Cloud Changes with `ctrl` + `shift` + `p`, I get an error. Attempting to enable using the UI results in the same as everyone above: nothing happens

![image](https://github.com/VSCodium/vscodium/assets/14098845/6e6a9d18-f912-4db6-8954-9fbb5f330de8)

VSCodium version 1.82.2

OS Info

![image](https://github.com/VSCodium/vscodium/assets/14098845/23634a67-32a5-41fa-b7ab-c2914af01ccb)

**Comment by yiannis-had** [comment]
Doesn't work for me either on:

Version: 1.82.2
Release: 23257
Commit: a5004432af6600c99fa4f918a3f2f8df44df71e6
Date: 2023-09-14T18:27:11.315Z
Electron: 25.8.1
ElectronBuildId: undefined
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Linux x64 6.4.6-76060406-generic

**Comment by mircsicz** [comment]
+1 for me neither on MacOS:

```
Version: 1.83.1
Release: 23285
Commit: 36d13de33ac0d6c684f10f99cff352e2f58228b3
Date: 2023-10-12T18:35:41.174Z (2 wks ago)
Electron: 25.8.4
ElectronBuildId: undefined
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin x64 22.6.0
```

nor Fedora 38 to which I currently don't have access

**Comment by daiyam** [comment]
#1709
